model-optimizer/extensions/ops/activation_ops.py

```diff
@@ -233,13 +233,13 @@ class Log(Activation):
 class SoftPlus(Activation):
     op = 'SoftPlus'
     version = 'opset4'
-    operation = staticmethod(lambda x: np.ln(np.exp(x) + 1.0))
+    operation = staticmethod(lambda x: np.log(np.exp(x) + 1.0))
```

Comment:
Please, add unit tests for the value propagation functions

Reply:
Tests for SoftPlus & Mish operations added